lib/MetaCPAN/Document/Release.pm

```diff
@@ -591,5 +592,77 @@ sub requires {
     };
 }
 
+sub _activity_filters {
+    my ( $self, $params, $start ) = @_;
+    my ( $author, $distribution, $module, $f )
+        = @{$params}{qw( author distribution module f )};
+
+    my @filters
+        = ( { range => { date => { from => $start->epoch . '000' } } } );
```

Comment:
this is from the original code in WEB, dating back to 2011 - first commit to the activity controller... of course no explanation there :)